2020-InfiniteRecharge/src/main/java/frc/robot/Robot.java

```diff
@@ -7,38 +7,36 @@
 
 package frc.robot;
 
-
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.SparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
```

Comment:
We don't use these imports in Robot.java - they should be removed. (We never declare any variables or function parameters of type `SparkMax` or the other types).